habitat_sim/robots/mobile_manipulator.py

```diff
@@ -91,13 +99,21 @@ class MobileManipulatorParams:
     arm_mtr_vel_gain: float
     arm_mtr_max_impulse: float
 
-    wheel_mtr_pos_gain: float
-    wheel_mtr_vel_gain: float
-    wheel_mtr_max_impulse: float
-
     base_offset: mn.Vector3
     base_link_names: Set[str]
 
+    wheel_joints: Optional[List[int]] = None
+    leg_joints: Optional[List[int]] = None
+    leg_init_params: Optional[List[float]] = None
+
+    wheel_mtr_pos_gain: Optional[float] = None
+    wheel_mtr_vel_gain: Optional[float] = None
+    wheel_mtr_max_impulse: Optional[float] = None
+
+    leg_mtr_pos_gain: Optional[float] = None
+    leg_mtr_vel_gain: Optional[float] = None
+    leg_mtr_max_impulse: Optional[float] = None
```

Comment:
I wonder if we should split the MobileManipulatorParams class apart to put the wheel stuff in the Fetch robot wrapper and the leg stuff in the Spot robot wrapper. I worry that this params class will keep growing in size as we add new robots.